mozilla/blurts-server
# CSAT survey for Petition banner experiment(MNTOR-3224)

# References:

Jira: [MNTOR-3224](https://mozilla-hub.atlassian.net/browse/MNTOR-3224)



# Description

Adds the CSAT banner for the “Petition banner experiment” introduced by PR #4711.

# How to test

1. (Re)build experiments: npm run build-nimbus
2. Navigate to the dashboard
3. US users should see the petition banner locally: Free users on the “action needed” and Plus users on the “fixed” tab.
4. After dismissing the petition banner and revisiting the respective dashboard tab the follow-up CSAT survey should show.

# Checklist (Definition of Done)



[MNTOR-3224]: https://mozilla-hub.atlassian.net/browse/MNTOR-3224?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
